src/core/particle_data.cpp

```diff
@@ -780,9 +775,16 @@ constexpr Utils::Vector3d ParticleProperties::rinertia;
 #endif
 
 #ifdef ROTATION
-void set_particle_rotation(int part, int rot) {
-  mpi_update_particle_property<uint8_t, &ParticleProperties::rotation>(part,
-                                                                       rot);
+void set_particle_rotation(int part, Utils::Vector3i const &flag) {
```

Comment:
Is Vector3bool possible?

Reply:
I guess so, but Cython doesn't like templated classes, so we would need to first define a `Vector3b` in the core, copy paste its struct definition in `utils.pxd`, and then deal with `bool` being a keyword and not a type in Cython.

Reply:
ok. too much hassle for 2 usecases